### ISSUE TITLE
Waters bridging between chains 

### ISSUE THREAD

**Issue by jamesmkrieger** [prody/ProDy#1987]
From ccp4bb:

Is there a something available to automatically analyse/identifiy/quantify bound (“bridging”) water molecules at protein-protein interfaces?  Could not find anyting right away.

**Comment by karolamik13** [comment]
No, but doing such a function is a very good idea. I will definitely think about it and add it when I have some time. I will let you know.
